uobtheatre/productions/schema.py

```diff
@@ -66,12 +66,17 @@ class ProductionNode(GrapheneImageMixin, DjangoObjectType):
     start = graphene.DateTime()
     end = graphene.DateTime()
 
+    is_bookable = graphene.Boolean()
```

Comment:
I think might make it an exclimation mark
```suggestion
    is_bookable = graphene.NonNull(graphene.Boolean())
```

Reply:
Or maybe even
```suggestion
    is_bookable = graphene.Boolean(required=True)
```